File: Quiz.Android/lib/models/option.dart

class Option {
  final int optionId;
  final String title;
  final bool isActive;
  final bool isCorrect;

  const Option({
    required this.optionId,
    required this.title,
    required this.isActive,
    required this.isCorrect,
  });

  factory Option.fromJson(Map<String, dynamic> json) {
    return Option(
      optionId: json['optionId'],
      title: json['title'],
      isActive: json['isActive'],
      isCorrect: json['isCorrect'],
    );
  }
}

File: Quiz.Entity/Models/Option.cs
using System.ComponentModel.DataAnnotations;

namespace Quiz.Entity.Models
{
    public class Option
    {
        [Key]
        public int OptionId { get; set; }
        public string Title { get; set; }
        public bool IsActive { get; set; }
        public bool IsCorrect { get; set; }

        public int QuestionId { get; set; }
    }
}
